tests/src/ComparisonUtils.h

```diff
@@ -9,11 +11,19 @@
 #include "UTIL/LCIterator.h"
 #include "EVENT/LCCollection.h"
 
+#include "podio/RelationRange.h"
+#include "podio/ObjectID.h"
+
 #include <cmath>
 #include <array>
 #include <iostream>
 #include <vector>
```

Comment:
```suggestion
#include <vector>
#include <algorithm>
```

Necessary for `std::count_if`